### PR TITLE
final layout

### PR DESCRIPTION
**Pull request by effie531**

### CODE REVIEW

**Comment by effie531** [comment]
merging final layout

File: index.html [hunk 1]
```diff
@@ -42,13 +42,13 @@ <h1> Need a Header </h1>
...
               Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. 
             </p>
-            <a href="">
-            <button>
-              Human Hi there 
-            </button>
-          </a> 
+
+            <div>
+            <a href= "link"> Intro- Human Hi There </a>
```
Comment (micposso) [inline index.html]: add title tag to links

File: index.html [hunk 2]
```diff
@@ -42,13 +42,13 @@ <h1> Need a Header </h1>
...
+            </div>
 
            </div>
+
            <figure>
              <img src="Images/manbaloon.jpeg" type= "Images" >
```
Comment (micposso) [inline index.html]: make the img tag self-closing

File: index.html [hunk 3]
```diff
@@ -63,33 +63,126 @@ <h1  Funny Hi there >
...
+              
+            </div>
+
+      </section> 
+
       <!-- Footer -->
-      <footer></footer>
+      <footer>
+        
+
```
Comment (micposso) [inline index.html]: remove white space

**Review by micposso — COMMENTED** [review]
good job, look at the comments, fix, commit and merge to master.

**Comment by effie531** [comment]
merged corrections suggested by Michael

**Pull request merged**